src/main/java/core/basesyntax/services/DataConverterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.services;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        return data.stream().skip(1)
+                .map(line -> {
+                    String[] parts = line.split(",");
+                    FruitTransaction.Operation operation
+                            = FruitTransaction.Operation.getByCode(parts[0].trim());
+                    String fruit = parts[1].trim();
+                    int quantity = Integer.parseInt(parts[2].trim());
+                    return new FruitTransaction(operation, fruit, quantity);
+                }).collect(Collectors.toList());
+    }

```

Comment:
While the method `convertToTransaction` is correctly converting data to transactions, consider adding error handling. If the data format is incorrect (e.g., not enough parts after splitting, or the quantity is not a valid integer), the method will throw an exception which might not be informative. It's better to catch such cases and throw a custom exception with a clear message indicating what went wrong and on which line.